### PR TITLE
First draft for sm api doc

### PR DESCRIPTION
**Pull request by mphprogrammer**
@Kam @ayaleloehr This is the format I'd like to follow. Please review, and if acceptable, I will have the rest up tonight.

### CODE REVIEW

**Comment by saneshark** [comment]
@mphprogrammer as long as we keep it consistent with what education claims and prescriptions has done already sounds good to me. 

Stick to consistency with prescriptions maybe since ts most of the same folks and they are the audience.

File: docs/sm-api.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* relationType: The relation between the veteran and the triage team.
+
+##### Getting a Triage Team List.
+To get a list of all triage teams assigned to the veteran the following HTTP VERB and URL are used:
+```
+GET /v0/messaging/health/recipients
+```
+An example response would be:
+```
+{"data":[{"id":"585968","type":"va_healthcare_messaging_triage_teams","attributes":{"triage_team_id":585968,"name":"Triage group 311070 test 1","relation_type":"PATIENT"}},{"id":"585986","type":"va_healthcare_messaging_triage_teams","attributes":{"triage_team_id":585986,"name":"Triage group 311070 test 2","relation_type":"PATIENT"}},{"id":"613586","type":"va_healthcare_messaging_triage_teams","attributes":{"triage_team_id":613586,"name":"Vets.gov Testing_DAYT29","relation_type":"PATIENT"}}]}
```
Comment (ayaleloehr) [inline docs/sm-api.md]: Can we better format the json so it's easier to read?

**Comment by ayaleloehr** [comment]
I think this works. I do wish we had consistency in how we documented APIs, but we just don't yet and would rather get something in as a starting point than argue about making it perfect. So i say keep going!

File: docs/sm-api.md (hunk 1 above)
Comment (carlmjohnson) [inline docs/sm-api.md]: Shout out to [json-tidy](https://github.com/carlmjohnson/json-tidy)…

Comment (carlmjohnson) [inline docs/sm-api.md]:
```json
{
	"data": [
		{
			"attributes": {
				"name": "Triage group 311070 test 1",
				"relation_type": "PATIENT",
				"triage_team_id": 585968
			},
			"id": "585968",
			"type": "va_healthcare_messaging_triage_teams"
		},
		{
			"attributes": {
				"name": "Triage group 311070 test 2",
				"relation_type": "PATIENT",
				"triage_team_id": 585986
			},
			"id": "585986",
			"type": "va_healthcare_messaging_triage_teams"
		},
		{
			"attributes": {
				"name": "Vets.gov Testing_DAYT29",
				"relation_type": "PATIENT",
				"triage_team_id": 613586
			},
			"id": "613586",
			"type": "va_healthcare_messaging_triage_teams"
		}
	]
}
```

**Comment by mphprogrammer** [comment]
@ayaleloehr @kam - there was only the one document in the docs folder - where is the prescriptions api document?

**Comment by ayaleloehr** [comment]
@mphprogrammer 

Rx still needs to be moved from the old repo here: https://github.com/department-of-veterans-affairs/prescriptions-api/blob/master/api-doc.md

Also, edu-benefits has an open PR for theirs: https://github.com/department-of-veterans-affairs/vets-api/pull/82/files

The document location for theirs (/docs/api/edu-benefits.md) is good to me as well, so maybe nesting this under /api would be good as well!

**Comment by ayaleloehr** [comment]
This is a good first draft. I think we will want some sample JSON responses in there, as I always like examples for making things tangible. But this is unequivocally better than what we had before, so going to merge.

**Pull request merged**